resources/lang/eng.yml

```diff
@@ -226,4 +228,7 @@ command-thru-usage: "//thru"
 command-unstuck-description: "Teleport to a safe location"
 command-unstuck-usage: "//unstuck"
 command-up-description: "Teleport up"
-command-up-usage: "//up <amount>"
\ No newline at end of file
+command-up-usage: "//up <amount>"
+
+command-iteminfo-description: "Get information about the item in your hand"
+command-iteminfo-usage: "//iteminfo"
```

Comment:
The lang command data should be in the same order as the commands are defined, so move these below the wand command